src/hooks/useAuth.ts

```diff
@@ -57,8 +55,8 @@ const useAuth = () => {
   };
 
   const logout = () => {
-    deleteCookie("accessToken");
-    deleteCookie("refreshToken");
+    deleteToken("accessToken");
```

Comment:
I think while logging out we can remove all session related data, so add a method and move these to the new method